core/templates/dev/head/components/forms/AudioTranslationsEditorDirective.js

```diff
@@ -106,6 +106,8 @@ oppia.directive('audioTranslationsEditor', [
 
                   // Whether there was an error uploading the audio file.
                   $scope.errorMessage = null;
+                  $scope.saveButtonText = 'Save';
```

Comment:
Are there any tests we can update/add to cover these changes?

Reply:
Er.. I don't think there are any tests to update.